process/peer/export_test.go

```diff
@@ -24,8 +24,6 @@ func (vs *validatorStatistics) GetMatchingPrevShardData(currentShardData block.S
 	return vs.getMatchingPrevShardData(currentShardData, shardInfo)
 }
 
-func (vs *validatorStatistics) PrevShardInfo() map[string]block.ShardData {
-	vs.mutPrevShardInfo.RLock()
-	defer vs.mutPrevShardInfo.RUnlock()
-	return vs.prevShardInfo
+func (vs *validatorStatistics) GetLeaderDecreaseCount(key []byte) uint32 {
+	return vs.missedBlocksCounters.get(key).leaderDecreaseCount
```

Comment:
is not mutex needed?

Reply:
Will add a mutex here and a concurrent unit test 👍

Reply:
this in export_test.go so only if you make the test concurrent it is needed. This is not in the production code.